staging/kos/pkg/controllers/ipam/ipam.go

```diff
@@ -888,12 +886,6 @@ func NewParsedLock(ipl *netv1a1.IPLock) (ans ParsedLock, err error) {
 	return
 }
 
-var _ fmt.Stringer = ParsedLock{}
-
```

Comment:
It was a deliberate choice to provide a more-concise-than-the-default result of `fmt.Sprintf("%s", aParsedLock)`.  Is there a reason to switch to the default?

Reply:
My apologies, I had checked the code and found that ParsedLocks were never printed like that, but I had missed this line: https://github.com/MikeSpreitzer/kube-examples/blob/make-subnets-validation-correct/staging/kos/pkg/controllers/ipam/ipam.go#L593. I reintroduced the custom `String()` function in b2e2d2b.